Classes/Backend/RecordList/RecordListConstraint.php

```diff
@@ -31,6 +32,11 @@ class RecordListConstraint
     public function isInAdministrationModule(): bool
     {
         $vars = GeneralUtility::_GET('route');
+        $version = (new Typo3Version())->getMajorVersion();
+        if ($version < 10) {
+            return $vars === '/web/NewsAdministration/';
+        }
+
```

Comment:
what about using str(r)pos() instead of a version check for this? would be much simpler here.

`return strpos('/web/NewsAdministration/', $vars) !== false;`